common/djangoapps/student/tests/test_create_account.py

```diff
@@ -278,6 +279,16 @@ def test_discussions_email_digest_pref(self, digest_enabled):
             else:
                 self.assertIsNone(preference)
 
+    def test_referral_attribution(self):
```

Comment:
Add docstrings for developers who come after us, and have no clue what attribution is.